src/Parser/BalanceDruid/CHANGELOG.js

```diff
@@ -1,3 +1,12 @@
 export default `
--
+22-09-2017 - Added Overcapped Lunar and Solar empowerments modules
+20-09-2017 - Minor fixes to Unemp LS module
+12-09-2017 - Added a module to track Unempowered LS casts
+12-09-2017 - Added the Damage module and Reorder of Stat boxes
+12-09-2017 - Minor text fixes
+07-09-2017 - Fixed stackable buffs - Now the ABC module should be more reliable
+05-09-2016 - Added Moons casted module
+04-09-2017 - Added Wasted Astral Power module
+02-09-2011 - Added Moonfire and Sunfire uptime modules
```

Comment:
2011 and 2016 eh? You've been working on this for a long time!

Reply:
Whats wrong with my brain today D: